mypy/fixup.py

```diff
@@ -123,6 +123,9 @@ def visit_type_var_expr(self, tv: TypeVarExpr) -> None:
             value.accept(self.type_fixer)
         tv.upper_bound.accept(self.type_fixer)
 
+    def visit_newtype_expr(self, nt: NewTypeExpr) -> None:
+        raise NotImplemented()
```

Comment:
That should be NotImplementedError(). But why raise at all? Most unimplemented functions here just fall back to the no-op base implemetation.